runtime/src/main/java/io/quarkiverse/cxf/CXFRecorder.java

```diff
@@ -30,7 +30,8 @@
                 String cfgSei = cxfEndPointConfig.serviceInterface.get();
                 if (cfgSei.equals(sei)) {
                     String endpointAddress = cxfEndPointConfig.clientEndpointUrl.orElse("http://localhost:8080");
-                    if (!relativePath.equals("/") && !relativePath.equals("")) {
+                    boolean clientEndpointAbsoluteUrl = cxfEndPointConfig.clientEndpointAbsoluteUrl.orElse(false);
```

Comment:
maybe if start with / it mean it is relative url and if start with scheme (http) it is absolute one ???